package.json

```diff
@@ -74,7 +78,18 @@
     "url": "git+https://github.com/jxnblk/styled-system.git"
   },
   "dependencies": {
-    "@babel/runtime": "^7.1.2",
+    "@babel/runtime": "^7.3.1",
     "prop-types": "^15.6.2"
```

Comment:
Probably not the best place to ask this, but have you considered dropping support for `prop-types`?

Reply:
Seems like an appropriate place to ask. What would the motivation for dropping prop-types be? I suspect these are still useful to people not using TS

Reply:
I prefer not to remove them because I'm currently using them for documenting available props in a [docz](http://docz.site/) site.

Reply:
> What would the motivation for dropping prop-types be?

They're no longer coupled with React and, as you said, superfluous to people using (TypeScript or Flow).  It's also pretty easy for a developer to keep them in a production build if they're not using something like `next` or `create-react-app`.